src/components/map/GroupMembers.tsx

```diff
@@ -0,0 +1,108 @@
+import { SERVER_URL } from '@env';
+import { useAuthContext } from '@nightlight/src/contexts/AuthContext';
+import React, { useEffect, useState } from 'react';
+import {
+  SafeAreaView,
+  View,
+  Pressable,
+  Animated,
+  ScrollView,
+} from 'react-native';
+import GroupMembersStyles from '@nightlight/components/map/GroupMembers.styles';
+import UserCircle from '@nightlight/components/map/UserCircle';
+import { Foundation } from '@expo/vector-icons';
+import { COLORS } from '@nightlight/src/global.styles';
+import { User } from '@nightlight/src/types';
+
+const GroupMembers = ({
+  userOnPress,
+  addGroupOnPress,
+}: {
+  /**
+   * Function to call when a user circle is pressed.
+   * @param userToShow User to show when circle is pressed.
+   */
+  userOnPress: (userToShow?: User) => void;
+  /**
+   * Function to call when the add button is pressed.
+   */
+  addGroupOnPress: () => void;
+}) => {
```

Comment:
Please export this into interfaces ts file

Reply:
Unless this is what we are doing now, I have no preference as long as it is consistent